lib/about_yml/schema.json

```diff
@@ -6,127 +6,127 @@
+        "milestones": {
+            "type": "array",
+            "description": "What are the key milestones you've achieved recently?",
+            "items": {"type": "string"},
+            "uniqueItems": true
         },
-        "stage": {
+        "owner_type": {
             "type": "string",
-            "enum": ["discovery", "alpha", "beta", "live", "sunset", "transfer", "end"],
-            "description": "Maturity stage of the project"
+            "description": "What kind of team owns the repository?",
+            "enum": ["guild", "working-group", "project"]
         },
-        "description": {
+        "stage": {
             "type": "string",
-            "description": "Description of the project"
-        },
-        "tags": {
-            "type": "array",
-            "description": "Tags that describe the project or aspects of the project",
-            "items": {"type": "string"},
-            "uniqueItems": true
+            "description": "MWhat is your project's current status?",
+            "enum": ["discovery", "alpha", "beta", "live"]
         },
         "testable": {
             "type": "boolean",
             "default": false,
             "description": "Should this repo have automated tests? If so, set to `true`."
         },
-        "team": {
-            "type": "array",
-            "description": "Team members contributing to the project",
-            "items": {
-                "$ref": "#/definitions/person"
-            },
-            "uniqueItems": true
+        "licenses": {
+            "type": "object",
+            "description": "What are the licences that apply to the project and/or its components?",
+            "patternProperties": {
+                ".*": {"$ref": "#/definitions/license"}
+            }
         },
         "partners": {
             "type": "array",
-            "description": "Partners for whom the project is developed",
+            "description": "Who is the parter for your project? (Use the full name of the partner documented here: https://github.com/18F/dashboard/blob/staging/_data/partners.yml)",
             "items": {"type": "string"},
             "uniqueItems": true
         },
-        "users": {
+        "contact": {
             "type": "array",
-            "description": "Organizations or individuals who have adopted the project for their own use",
+            "description": "The main point of contact(s) for your project, and a `mailto:` link for each contact.",
             "items": {
-                "$ref": "#/definitions/user"
+                "$ref": "#/definitions/contact"
             },
             "uniqueItems": true
         },
-        "milestones": {
+        "team": {
             "type": "array",
-            "description": "Brief descriptions of significant project developments",
-            "items": {"type": "string"},
+            "description": "Who are the team members on your project?",
```

Comment:
Here we can add something like: "You can specify GitHub usernames, email addresses, or other organizational usernames."